src/winter_onboarding_2021/fleet_management_service/specs.clj

```diff
@@ -24,3 +24,20 @@
   (s/keys :req [:cabs/name
                 :cabs/licence-plate
                 :cabs/distance-travelled]))
+
+;; Users
+(s/def :users/name string?)
```

Comment:
[Naming] Do you think it's better to use singular for the spec names? For example: `:user/name` since the name is on a single user, and not a group of users. 

In the same spirit, naming the `users` spec to `user`

Reply:
We named it that way because of the table names in db. JDBC namespaces the keys by the tablename.